packages/shared/src/components/auth/AuthExtensionBanner.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { BottomBannerContainer } from '../banners';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import { MemberAlready } from '../onboarding/MemberAlready';
+import { AuthTriggers } from '../../lib/auth';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { GetExtensionButton } from '../buttons/GetExtensionButton';
+
+export const BottomBannerBg =
+  "bg-cover bg-[url('https://daily-now-res.cloudinary.com/image/upload/s--lf8LUJjq--/f_auto/v1732012913/login-popover-dailydev_mxb7lw')]";
```

Comment:
For the arbitrary value, we can do an inline `style` prop on the component. I think that is much easier to read. Non blocking though.